.github/ISSUE_TEMPLATE.md

```diff
@@ -43,27 +43,27 @@ be fixed in the latest versions. -->
 
 **firebase-admin:**
 
-### Test case
+### [REQUIRED] Test case
 
 <!-- Provide a minimal, complete, and verifiable example (http://stackoverflow.com/help/mcve) -->
 
 
-### Steps to reproduce
+### [REQUIRED] Steps to reproduce
 
 <!-- Provide the steps needed to reproduce the issue given the above test case. -->
 
 
-### Were you able to successfully deploy your functions?
+### [REQUIRED] Were you able to successfully deploy your functions?
 
 <!-- When you ran `firebase deploy`, did you see any error messages? -->
```

Comment:
Should this one be required? Or have a comment that they should put 'n/a'?

Reply:
Good point - I'm going to remove the require here.